repose-aggregator/docs/build.gradle

```diff
@@ -0,0 +1,286 @@
+import org.ajoberstar.grgit.Grgit
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.regex.Pattern
+
+apply plugin: 'org.openrepose.gradle.plugins.linkchecker'
+
+def username = System.properties['user.name']
+def uid = ["id", "-u", username].execute().text.trim()
+def gid = ["id", "-g", username].execute().text.trim()
+
+task asciibinder(group: 'build', description: 'Builds the latest version of the docs') {
+    inputs.dir(file("${project.projectDir}/src"))
+    outputs.dir(file("${project.buildDir}/asciibinder"))
+    doLast {
+        copy {
+            from(fileTree("${project.projectDir}/src/asciibinder"))
+            into("${project.buildDir}/asciibinder")
+        }
+        copy {
+            from(fileTree("${project.projectDir}/src/templates"))
+            into("${project.buildDir}/asciibinder")
+            expand(version: version)
+        }
+
+        def grgit = Grgit.init(dir: "${project.buildDir}/asciibinder")
+        grgit.add(patterns: ['.'])
+        grgit.commit(message: 'initial_commit')
+
+        def buildProcess =
+            "docker build --tag local/ascii_binder ${project.projectDir}/src/docker/AsciiBinder/"
```

Comment:
Since we are now in full control of this image, I wasn't as worried about cleaning it up. It isn't super huge and it shaves a little off the build time to keep it around.